### PR TITLE
Test ci

### PR DESCRIPTION
**Pull request by robertosilvino**

### CODE REVIEW

File: classes/event/relationship_created.php [hunk 1]
```diff
@@ -40,6 +40,7 @@ class relationship_created extends \core\event\base {
     protected function init() {
         $this->data['crud'] = 'c';
         $this->data['level'] = self::LEVEL_OTHER;
+        $this->data['edulevel'] = self::LEVEL_OTHER;
```
Comment (robertosilvino) [inline classes/event/relationship_created.php:43]: !resolve

Comment (robertosilvino) [inline classes/event/relationship_created.php:43]: testar !fix

Comment (robertosilvino) [inline classes/event/relationship_created.php:43]: !fixed

File: classes/event/relationship_deleted.php [hunk 2]
```diff
@@ -39,6 +39,7 @@ class relationship_deleted extends \core\event\base {
     protected function init() {
         $this->data['crud'] = 'd';
         $this->data['level'] = self::LEVEL_OTHER;
+        $this->data['edulevel'] = self::LEVEL_OTHER;
```
Comment (robertosilvino) [inline classes/event/relationship_deleted.php:42]: !fix

Comment (robertosilvino) [inline classes/event/relationship_deleted.php:42]: !fixed

File: classes/event/relationshipgroup_member_added.php [hunk 3]
```diff
@@ -40,6 +40,7 @@ class relationshipgroup_member_added extends \core\event\base {
     protected function init() {
         $this->data['crud'] = 'c';
         $this->data['level'] = self::LEVEL_OTHER;
+        $this->data['edulevel'] = self::LEVEL_OTHER;
```
Comment (robertosilvino) [inline classes/event/relationshipgroup_member_added.php:43]: !fix

Comment (robertosilvino) [inline classes/event/relationshipgroup_member_added.php:43]: !completed

**Comment by caiodoneda** [comment]
olá

**Comment by robertosilvino** [comment]
oi

**Comment by arthurd2** [comment]
ola, quer tc?

On 13/06/2016 10:05, robertosilvino wrote:

> oi
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/UFSC/moodle-local-relationship/pull/1#issuecomment-225574904, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/ACUaZjiLYBNx63YHac9RoQ--Y2CLyMwZks5qLVWbgaJpZM4IsL09.

## 

[]'s
Arthur

**Pull request merged**